src/geff/spatial_graph/io.py

```diff
@@ -125,14 +131,12 @@ def read_sg(
         A SpatialGraph containing the graph that was stored in the geff file
         format.
     """
-    # zarr python 3 doesn't support Path
-    path = str(path)
 
     # open zarr container
     if validate:
-        geff.utils.validate(path)
+        geff.utils.validate(store)
 
-    group = zarr.open(path, mode="r")
+    group = zarr.open(store, mode="r")
```

Comment:
Can you add the expand tilde thing here too? We need it, right, since it isn't using GeffReader (yet..)?